### ISSUE TITLE
Unable to deploy war file

### ISSUE THREAD

**Issue by ctproject4** [Infosys/openIDP#25]
I'm trying to build and deploy maven project. Build Job is running properly after triggering the pipeline and it also creates .war file. The deployment Job also seems normal and executing successful but after opening the url in tomcat server **404 Page Not Found** error use to come. I've tried to deploy the war file on the **same tomcat server in which jenkins is running**. Also I tried to deploy it onto another machine as well. But still no luck. I'm attaching the log files of Deploy Job as well as Pipeline Job. Kindly help me!

[Deploy_Job_Logs.txt](https://github.com/Infosys/openIDP/files/2690817/Deploy_Job_Logs.txt)
[Piepline_logs.txt](https://github.com/Infosys/openIDP/files/2690818/Piepline_logs.txt)

![jenkins-page](https://user-images.githubusercontent.com/36787361/50159180-1216a980-02fc-11e9-924d-64ca84ede519.PNG)

 Thanks

**Comment by jaskirat8** [comment]
@ctproject4  It will be helpful if you can attach screen shot of your "Deploy Page" and "Code Page" in IDP UI. As i can see there is nothing configured as per logs.

**Comment by ctproject4** [comment]
Thanks for your reply. I'm attaching the screenshots.
![deploypage2](https://user-images.githubusercontent.com/36787361/50336975-ad469380-0535-11e9-8b1d-63878454e532.PNG)
![code_page](https://user-images.githubusercontent.com/36787361/50336977-ad469380-0535-11e9-89f5-8e40c57acdc6.PNG)
![deploy_page1](https://user-images.githubusercontent.com/36787361/50336979-ad469380-0535-11e9-9d97-f63ecbe9a9c5.PNG)
Thanks!

**Comment by Chids-gs** [comment]
@ctproject4 . Few suggestions
1. can you check in your workspace and confirm if the war file is generated?
2. If yes, in the deploy section, can you replace the war file path to **/*.war

**Comment by ctproject4** [comment]
@Chids-gs Yes the workspace containes .war file after build stage. And I also tried using target/**/*.war.
Still no luck!

**Comment by snj07** [comment]
Can you confirm if tomcat user which you put in the input field has manager-script permission?

**Comment by jaskirat8** [comment]
@ctproject4 We hope you have tried the suggestions by @Chids-gs @snj07 .
Update this thread in-case you still face the issue. 
- This issue will auto close in 3 days on account of no activity.

**Comment by ctproject4** [comment]
@jaskirat8 Yes I tried the suggestions given by @snj07 and @Chids-gs. Still no luck!

**Comment by krishnakanthbn** [comment]
@ctproject4 Hi. Hope the issue is resolved. Can you please confirm if you see the Tomcat deploy section in the Deploy Job configuration in Jenkins?
